examples/async_await/bin/futures_intro.dart

```diff
@@ -0,0 +1,27 @@
+// #docregion ''
+Future<void> getUserOrder() =>
+    // Imagine that this function fetches user info from a data source
+    Future.delayed(
```

Comment:
I was surprised to see => used for > one line of code.
(I looked it up in Effective Dart, and I don't see anything saying
you should stick to one line for =>... but still, it surprised me.)
[Old code](https://dart.dev/codelabs/async-await#example-incorrectly-using-an-asynchronous-function)
[New code](https://dart-dev-staging-1.firebaseapp.com/codelabs/async-await#example-incorrectly-using-an-asynchronous-function)

/cc @munificent

Reply:
There are no hard and fast rules for how long the body of an `=>` member can be. I think once it gets to be more than a few lines, then a regular braced body is easier to read (and has less indentation), but it's mostly a judgement call.

Reply:
For me, the `=>` made this sample harder to read. It's easy to miss the `=>` and wonder where the `}` went.

Reply:
I've reverted to using `{...}`.